Python/Product/TestAdapter/testlauncher.py

```diff
@@ -12,24 +30,37 @@ def parse_argv():
     2. Test runner, `pytest` or `nose`
     3. debugSecret
     4. debugPort
-    5. Rest of the arguments are passed into the test runner.
+    5. Debugger path
+    6. Rest of the arguments are passed into the test runner.
     """
 
-    return (sys.argv[1], sys.argv[2], sys.argv[3], int(sys.argv[4]), sys.argv[5:])
+    return (sys.argv[1], sys.argv[2], sys.argv[3], int(sys.argv[4]), sys.argv[5], sys.argv[6:])
 
-
-def exclude_current_file_from_debugger(secret, port):
+def load_debugger(cwd, secret, port, debugger_search_path):
     # Load the debugger package
     try:
-        import ptvsd
+        sys.path[0] = os.getcwd()
+        os.chdir(cwd)
```

Comment:
chdir is needed for running the test, regardless of debugging. I feel it would be better to move that up to main.